multivac/gather_data.py

```diff
@@ -105,6 +106,16 @@ def github_time_to_unix(time: str) -> float:
     return time_to_unix
 
 
+# https://github.com/tarantool/tarantool/tree/master/.github/workflows
+os_matcher = re.compile(r"[a-z]+_[0-9]+(_[0-9]+)?")
+freebsd_matcher = re.compile(r"freebsd-[0-9]{2}")
+"""Jobs running right on the runner, not in a container"""
```

Comment:
I think it will be ok if it is a normal Python comment.

Reply:
fixed